docs/dispatcher/filters/command.rst

```diff
@@ -18,10 +18,10 @@ When filter is passed the :class:`aiogram.filters.command.CommandObject` will be
 Usage
 =====
 
-1. Filter single variant of commands: :code:`Command(commands=["start"])` or :code:`Command(commands="start")`
-2. Handle command by regexp pattern: :code:`Command(commands=[re.compile(r"item_(\d+)")])`
-3. Match command by multiple variants: :code:`Command(commands=["item", re.compile(r"item_(\d+)")])`
-4. Handle commands in public chats intended for other bots: :code:`Command(commands=["command"], commands_ignore_mention=True)`
+1. Filter single variant of commands: :code:`Command("start")`
```

Comment:
probably add here example of `CommandStart`?

Reply:
and `BotCommand`

Reply:
Yes.